pkg/cmd/server/api/types.go

```diff
@@ -1407,3 +1408,16 @@ type DefaultAdmissionConfig struct {
 	// Disable turns off an admission plugin that is enabled by default.
 	Disable bool
 }
+
+// TemplateServiceBrokerConfig holds information related to the template
+// service broker
+type TemplateServiceBrokerConfig struct {
+	// Enabled switches on the types, API and controllers associated with the
+	// template service broker
+	Enabled bool
+
+	// TemplateNamespaces indicates the namespace(s) in which the template service
+	// broker looks for templates to serve.  If empty, the default is the
```

Comment:
> So it's now mandatory to define the set of namespaces in the config instead of defaulting to the sane value everyone would expect? Why do we hate our users?

One person's "sane" is another person's "magic".  Choosing to set a default based on emptiness steals power from the API and (at least from the outside), its not obvious that this is as common as something like the "openshift" namespace.

In addition, you're going to have a bootstrapping problem with squatters.  If you have a default you're setting, you have to deal with the squatters. If you don't have a default, then you don't have to deal with it.